### PR TITLE
Various small fixes

### PR DESCRIPTION
**Pull request by maurycyp**
Note that `network_id = 0` was deleted because it is redundant.

### CODE REVIEW

File: pyethapp/jsonrpc.py [hunk 1]
```diff
@@ -839,10 +840,10 @@ class Filter(object):
     :ivar new_logs: same as :attr:`logs`, but is reset at every access
     """
 
-    def __init__(self, chain, blocks=[], addresses=None, topics=None,
+    def __init__(self, chain, blocks, addresses=None, topics=None,
```
Comment (heikoheiko) [inline pyethapp/jsonrpc.py]: are you sure, that it must not called without blocks?

File: pyethapp/synchronizer.py [hunk 2]
```diff
@@ -262,12 +262,16 @@ def receive_newblock(self, proto, t_block, chain_difficulty):
...
+                      expected_difficulty=expected_difficulty)
             self.chainservice.broadcast_newblock(t_block, chain_difficulty, origin=proto)
         else:
             # any criteria for which blocks/chains not to add?
             log.debug('insufficient difficulty, dropping',
-                      client=proto.peer.remote_client_version, chain_difficulty=chain_difficulty, expected_difficulty=expected_difficulty)
+                      client=proto.peer.remote_client_version,
+                      chain_difficulty=chain_difficulty,
+                      expected_difficulty=expected_difficulty)
 # return  # FIXME
```
Comment (heikoheiko) [inline pyethapp/synchronizer.py]: these were probably changed already. note, we have 99 chars line width.

Comment (maurycyp) [inline pyethapp/synchronizer.py]: OK, I was assuming 80.

File: pyethapp/jsonrpc.py (hunk 1 above)
Comment (maurycyp) [inline pyethapp/jsonrpc.py]: My mistake. That should be `blocks=None`, and then `blocks = blocks or []` below that.

Comment (heikoheiko) [inline pyethapp/jsonrpc.py]: looking at the code i actually think the signature should be:
`def __init__(self, chain, blocks=set(), addresses=set() topics=set(), pending=False, latest=False)`

combined with appropriate changes in the code following.

BUT: it also depends on how the code is called. i'd rather not change anything here without proper unit tests.

**Comment by heikoheiko** [comment]
i appreciate your updates. maybe you can create a PR w/o above issues. or with unit tests :)

File: pyethapp/jsonrpc.py (hunk 1 above)
Comment (maurycyp) [inline pyethapp/jsonrpc.py]: Yeah, my goal wasn't to change the signature, but to avoid the problematic mutable `[]`.

**Comment by maurycyp** [comment]
Sure, I'll probably create a new PR without the 2 problematic changes (unit tests might come later).  I don't think I can update the current PR because I already deleted my fork.

**Comment by maurycyp** [comment]
Resubmitted as https://github.com/ethereum/pyethapp/pull/10 .

**Pull request closed without merging**